src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,16 @@
 const accessibilityQuiz = [
+  {
+    {
+    "Question":
+    "What is the importance of web accessibility, and how does it benefit users and businesses?",
+    "Answer": "C) It ensures that websites and applications are usable by people with disabilities, improving user experience and inclusivity.",
+    "Distractor1": "A) It's not important; it doesn't benefit users or businesses.",
+    "Distractor2": "B) It ensures compatibility with the latest web technologies.",
+    "Distractor3": "D) It's mainly for legal compliance.",
```

Comment:
remember that the quiz randomly sorts answers each time the quiz starts so the A),B),C),D) here doesn't make sense

```suggestion
    "Answer": "It ensures that websites and applications are usable by people with disabilities, improving user experience and inclusivity.",
    "Distractor1": "It's not important and doesn't benefit users or businesses.",
    "Distractor2": "It ensures compatibility with the latest web technologies.",
    "Distractor3": "It's mainly for legal compliance.",
```